src/portal/components/NodeActionBtn.vue

```diff
@@ -130,15 +128,15 @@ export default class NodeActionBtn extends Vue {
       this.openUnreserveDialog = false;
     } else {
       this.$toasted.show(`unreserving node ${this.nodeIDToUnreserve}`);
-      const rentContractID = await getRentContractID(
-        this.$api,
-        this.nodeIDToUnreserve
-      );
+      const rentedNodes = await getRentedNodes()
+      const rentedNode = rentedNodes.filter((contractID: { nodeId: string; }) => contractID.nodeId === this.nodeIDToUnreserve);
+      console.log("rentContractID", rentedNode);
+            
```

Comment:
instead of getting all rented nodes, we can get this node by its id from gridproxy`https://gridproxy.dev.grid.tf/nodes/{nodeId}` and get the rent contractId